Films_json.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Pulp_Fiction",
+    "genre": "thriller",
+    "country": "US",
+    "releaseYear": 1994,
+     "durationMin": 154
```

Comment:
I like how you mention the measurement units here, makes sense as the value should be interpreted correctly